src/Microsoft.VisualStudio.Editors/PropPages/AdvBuildSettingsPropPage.vb

```diff
@@ -103,6 +85,19 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
         Protected Overrides Sub PreInitPage()
             MyBase.PreInitPage()
             PopulateLanguageVersions()
+            SetupLangVersionLinkLabel()
+        End Sub
+
+        ''' <summary>
+        ''' Sets up the link to take the user to more information on why they can't change the language
+        ''' version.
+        ''' </summary>
+        Private Sub SetupLangVersionLinkLabel()
+            Dim link = New LinkLabel.Link
+            link.LinkData = My.Resources.Strings.CantSelectLanguageVersionFWLink
+            lnkLabel.Links.Clear()
+            lnkLabel.Links.Add(link)
+            lnkLabel.Text = My.Resources.Strings.CantSelectLanguageVersionLinkLabel
```

Comment:
Why does this need to be dynamic every time the page is initialized? Is there a reason this can't be done in InitializeComponent?

Reply:
I'll investigate.

Reply:
I simplified this code and moved it so it is called once in the constructor.